Description: Find cairo.h
  Find cairo.h by adding cflags from pkg-config.
Author: Tobias Hansen <thansen@debian.org>

--- a/higan/GNUmakefile
+++ b/higan/GNUmakefile
@@ -23,7 +23,7 @@
     link += -dynamiclib
   endif
 else ifneq ($(filter $(platform),linux bsd),)
-  flags += -fopenmp
+  flags += -fopenmp $(shell pkg-config gtk+-2.0 --cflags)
   link += -fopenmp
   ifeq ($(binary),application)
     link += -Wl,-export-dynamic
